klab.hub/src/main/java/org/integratedmodelling/klab/hub/api/TokenVerifyEmailClickback.java

```diff
@@ -1,22 +1,40 @@
 package org.integratedmodelling.klab.hub.api;
 
+import org.integratedmodelling.klab.hub.config.LinkConfig;
 import org.springframework.data.annotation.TypeAlias;
 
 @TypeAlias("VerifyEmail")
 public class TokenVerifyEmailClickback extends TokenClickback {
 	
 	private static final long serialVersionUID = 2577854654763037014L;
+	private String notVerifiedEmail;
```

Comment:
Question about naming (we know that naming is hard 😅): `notVerifiedEmail` vs `newEmail` ?